packages/cfd/src/Containers/__tests__/compare-accounts-modal.spec.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import CompareAccountsModal from '../compare-accounts-modal';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('../mt5-compare-table-content', () => jest.fn(() => 'MockedMt5CompareTableContent'));
+
+describe('CompareAccountsModal', () => {
+    let modal_root_el;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    let mock_props;
+    beforeEach(() => {
```

Comment:
Hello @thisyahlen-deriv,

Do you need to reset the `mock_props` before each test?

Reply:
Yes sir, In some tests im using dxtrade, so i need that before each to reset to mt5